python/equistore-core/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+python_files = *.py
+testpaths = tests
```

Comment:
Why not putting this into the `pyproject.toml`?

Reply:
I can try, I think there was some issue with pytest rootdir without this file.

Reply:
no, it works!